projects/core/src/global-message/http-interceptors/handlers/bad-request.handler.ts

```diff
@@ -15,20 +15,17 @@ export class BadRequestHandler extends HttpErrorHandler {
   handleError(request: HttpRequest<any>, response: HttpErrorResponse): void {
     if (
       response.url.includes(OAUTH_ENDPOINT) &&
-      response.error.error === 'invalid_grant'
+      response.error.error === 'invalid_grant' &&
```

Comment:
We could move those hardcoded strings to consts, like with `OAUTH_ENDPOINT`

Reply:
let's do that in #2863